### PR TITLE
reverse_words: add --data_path, make save_path optional

### PR DESCRIPTION
**Pull request by amitbeka**
Signed-off-by: Amit Beka <amit.beka@gmail.com>

### CODE REVIEW

File: reverse_words/__main__.py [hunk 1]
```diff
@@ -24,12 +25,18 @@
              " In the `sample` and `beam_search` modes a trained model is "
              " to used reverse words in the input text.")
     parser.add_argument(
-        "save_path", default="chain",
+        "save_path", default="chain", nargs='?',
```
Comment (rizar) [inline reverse_words/__main__.py]: The default name "chain" is a remnant from the past of this  script. Could you please change it something more sensible? E.g. `reverse_words.zip`.

**Comment by amitbeka** [comment]
I changes it to 'reverse_words' without suffix because it creates multiple files (model and log)

**Comment by rizar** [comment]
Oops, I thought I had answered, that `reverse_words.zip` is preferable because `reverse_words` clashes with the directory name.

**Comment by rizar** [comment]
Thanks! A hint for the future: I do not receive any message when one makes a commit, so in order to notify me that something has been done, please post a reply here.

**Comment by amitbeka** [comment]
this explains it - thanks :)

**Pull request merged**